Makefile

```diff
@@ -0,0 +1,33 @@
+export GO111MODULE := on
```

Comment:
minor: afaik this is no longer necessary in new go versions (which I would target)